extension/spring-test/src/main/kotlin/org/camunda/bpm/extension/process_test_coverage/spring_test/ProcessEngineCoverageProperties.kt

```diff
@@ -0,0 +1,38 @@
+package org.camunda.bpm.extension.process_test_coverage.spring_test
+
+import org.assertj.core.api.Condition
+import org.camunda.bpm.extension.process_test_coverage.spring_test.ProcessEngineCoverageTestExecutionListener.Companion.DEFAULT_ASSERT_AT_LEAST_PROPERTY
+
+open class ProcessEngineCoverageProperties {
```

Comment:
Interesting decision for properties to be a class with no members but rather functions. Let's discuss this. Why not using a data class for it?

Reply:
Sometimes my long development in Java still blinds me. Started with data class, but somehow got off the track with this solution. Now implemented as a data class with a builder.